packages/model-viewer/src/three-components/gltf-instance/ModelViewerGLTFInstance.ts

```diff
@@ -185,21 +178,38 @@ export class ModelViewerGLTFInstance extends GLTFInstance {
       clone.normalMap = material.normalMap.clone();
       clone.normalMap.needsUpdate = true;
     }
-    if (material.aoMap != null) {
-      clone.aoMap = material.aoMap.clone();
-      clone.aoMap.needsUpdate = true;
-    }
     if (material.emissiveMap != null) {
       clone.emissiveMap = material.emissiveMap.clone();
       clone.emissiveMap.needsUpdate = true;
     }
-    if (material.metalnessMap != null) {
+
+    const roughnessMap: Texture|null = material!.roughnessMap!.clone();
```

Comment:
nit: don't think you need the first "!".

Reply:
done.